resources/scripts/components/server/Console.tsx

```diff
@@ -82,26 +88,34 @@ export default () => {
     useEffect(() => {
         if (connected && ref.current && !terminal.element) {
             terminal.open(ref.current);
+            terminal.loadAddon(fitAddon);
+            terminal.loadAddon(searchAddon);
+            terminal.loadAddon(searchAddonBar);
+            fitAddon.fit();
 
-            // @see https://github.com/xtermjs/xterm.js/issues/2265
-            // @see https://github.com/xtermjs/xterm.js/issues/2230
-            TerminalFit.fit(terminal);
-
-            // Add support for copying terminal text.
+            // Add support for capturing keys
             terminal.attachCustomKeyEventHandler((e: KeyboardEvent) => {
-                // Ctrl + C
+            // Ctrl + C ( Copy )
                 if (e.ctrlKey && (e.key === 'c')) {
                     document.execCommand('copy');
                     return false;
                 }
 
+                if (e.ctrlKey && (e.key === 'f')) {
```

Comment:
those parentheses are unnecessary, I'd remove them unless the linter complains otherwise.

Reply:
I was just following what was already there, if you have an improvement, please do post.

Reply:
Not quite sure why I put those there in the first place (for line 99), they can both be removed.

Reply:
The issue was you couldn't Ctrl-C with selected text in the console, as it was sending ctrl-c to the console itself.

Reply:
https://github.com/pterodactyl/panel/commit/b05048871c65141b64df0912024f1bfef4c3527b

Reply:
I was just referring to code style, the following should be equivalent:
```suggestion
                if (e.ctrlKey && e.key === 'f') {
```